freeipa/src/main/java/com/sequenceiq/freeipa/retry/RetryService.java

```diff
@@ -0,0 +1,20 @@
+package com.sequenceiq.freeipa.retry;
+
+import org.springframework.retry.annotation.Backoff;
+import org.springframework.retry.annotation.Retryable;
+import org.springframework.stereotype.Service;
+
+import com.sequenceiq.cloudbreak.util.CheckedFunction;
+import com.sequenceiq.freeipa.client.RetryableFreeIpaClientException;
+
+@Service
+public class RetryService {
```

Comment:
I don't know if you are aware of, but we already have a retryservice in thhe common. You might rename this just to avoid confusion later